### ISSUE TITLE
Cannot update packages

### ISSUE THREAD

**Issue by RefusesNames** [wbthomason/packer.nvim#82]
I can install new plugins, but when I use `:PackerUpdate` every plugin is marked as `Failed to update <PluginName>`. Is there a way to see the actual error? The messages at the bottom say something about `rplugin.vim`, but  this file is empty:
```
" perl plugins


" node plugins


" python3 plugins


" ruby plugins


" python plugins

```

**Comment by wbthomason** [comment]
What does `:messages` show, exactly?

**Comment by RefusesNames** [comment]
```
remote/host: generated rplugin manifest: C:\Users\<UserName>\AppData\Local\nvim-data/rplugin.vim
```
Not sure if that helps though

**Comment by wbthomason** [comment]
Hmm, ok. Do you get anything when you press enter on the "Failed" lines?

**Comment by RefusesNames** [comment]
```
 ✗ Failed to update loricandre\fzterm.nvim
  Errors:
    5ceb7ca
    5ceb7ca
 ✗ Failed to update wbthomason\packer.nvim
  Errors:
    8dc961b
    8dc961b
 ✗ Failed to update nvim-lua\completion-nvim
  Errors:
    c059416
    c059416
 ✗ Failed to update thiagoalessio\rainbow_levels.vim
  Errors:
    ee43f5b
    ee43f5b
```
Not really anything helpful

**Comment by wbthomason** [comment]
Ok. I should probably look into adding more logging so that people can get better information in the case of failures.

Without said additional information, I'm not sure why updates don't work for you (since, for the vast majority of users, they seem to work correctly)....

Do you happen to have a minimal config that reproduces the behavior?

**Comment by RefusesNames** [comment]
I can try to build a minimal example later, but for now I only have my full setup [here](https://github.com/RefusesNames/vim_config/tree/fzterm) (using `NVIM v0.5.0-808-gee3acef38` on Windows).

**Comment by wbthomason** [comment]
Hmm. I wonder if this is a Windows thing (which would be unfortunate because I don't have a Windows machine to try and repro errors).

What's your `git` version, out of curiosity? There have been issues previously with `git` versions on Windows.

**Comment by RefusesNames** [comment]
I do not know what happened in this time, but it now works. There are too many variables to pin point what fixed it (I updated Neovim, I updated Packer, I switched the source of my git binaries and updated on the way). My guess is that switching to a newer version of git provided via scoop did the trick. If anyone runs into the same issue, I would suggest you start there. 

For me it's now fixed.
